backend/src/main/java/com/project/G1_T3/leaderboard/model/LeaderboardPlayerProfile.java

```diff
@@ -18,21 +18,21 @@ public class LeaderboardPlayerProfile {
     UUID profileId;
     String firstName;
     String lastName;
-    float ELO;
+    int glickoRating;
     Long position;
 
-    public LeaderboardPlayerProfile(UUID profileId, String firstName, String lastName, Float eLO) {
+    public LeaderboardPlayerProfile(UUID profileId, String firstName, String lastName, int glickoRating) {
         this.profileId = profileId;
```

Comment:
why the spelling of eLO like that HAHAHA

Reply:
wait thanks for changing it to glickoRating though.